citizennet/cn-batch-forms
# API-2850 Refactor Batch-Edit Form Operation Defaults

## Jira Ticket

- [https://citizennet.atlassian.net/browse/API-2850](https://citizennet.atlassian.net/browse/API-2850)

## What does this pull request do?

This PR resolves an issue wherein the default batch operations were not being selected for certain fields on the batch-edit form. I don't have much of an explanation as to why this was breaking things, but the solution works. I had click-tested the batch-edit form quite a lot to ensure there weren't any side effects.

## How should this be manually tested?

1. Select two or more ad sets/campaigns.
2. Navigate to the [batch-edit form](https://jh.aws.citizennet.com/manager/#/groups/facebook/168260/campaigns/572011/ad-sets/3055312:3055313/details?debug).
3. Ensure the `Replace` operation is pre-selected for the `Facebook Positions` dropdown.
